gallery/src/components/NxThreatCounter/NxThreatCounterPage.tsx

```diff
@@ -22,7 +22,10 @@ const NxThreatCounterPage = () =>
     <GalleryDescriptionTile>
       <p className="nx-p">
         Component for a series of small counters which display a number representing a count of threats/violations,
-        and a short pre-set text string which displays the severity (critical, severe, moderate, and low).
+        and a short pre-set text string which displays the severity (critical, severe, moderate, low, and none).
+      </p>
+      <p className="nx-p">
+        Each count is optional. If no counts are provided, then nothing will be rendered.
```

Comment:
I'd rephrase this a little bit, as using this component to render nothing at all wouldn't really be a useful/supported case.  How about this?
```suggestion
        Each count is optional. If no value is provided for a given count, then the indicator for that severity level will not be rendered.
```